database/create_table.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS user_language;
+CREATE TABLE  user_language(
+    user_id INT NOT NULL,
+    language_from VARCHAR(100),
```

Comment:
hmm do you need that many characters?

Reply:
by the way you can only store 2 characters, that means you can use char instead of varchar